Monika-After-Story/MonikaModDev
# Better Checkbox UI

# Key Changes
* Adds a "checkbox" to the `check_scrollable_menu` that gets filled when an item is **selected**
* Adds a `generic_fancy_check_button` that can be used in place of `check_button` in places that float text - namely the Outfit Mode checkbox. This style of checkbox colors the entire frame when the item is selected/hovered, making it obvious when the mouse is over the checkbox and when the checkbox is selected.
* Updated the test check scrollable menu so it uses the updated UI areas
* Added additional check scrollable menu dev label that just shows a sampling of buttons (Sample Check Scrollable Menu)

### Dark Mode
![darkmodecheckboxbuttons](https://user-images.githubusercontent.com/3499462/91531824-081bbb00-e8d3-11ea-8e12-ea91dc57bff5.png)

### Light Mode
![lightmodecheckboxbuttons](https://user-images.githubusercontent.com/3499462/91531850-0d790580-e8d3-11ea-8cb9-f66d64ab739a.png)

### outfit mode checkbox (both light and dark mode)
![outfitmode_changes](https://user-images.githubusercontent.com/3499462/91532046-56c95500-e8d3-11ea-8b9d-ef2f6b31c07d.gif)

# Testing
* verify check scrollable menus look okay
* verify Outfit Mode checkbox looks okay
